Integrations/PolySwarm/PolySwarm.py

```diff
@@ -269,7 +269,10 @@ def test_connectivity(self):
         return True
 
     def file_reputation(self, param):
-        title = 'PolySwarm File Reputation for Hash: %s' % param['hash']
+        file_hash = param.get('hash', param.get('file'))
```

Comment:
I think its better to switch it

```suggestion
        file_hash = param.get('file', param.get('hash'))
```